openapi3/loader.go

```diff
@@ -683,6 +683,24 @@ func (loader *Loader) resolveResponseRef(doc *T, component *ResponseRef, documen
 		sort.Strings(examples)
 		for _, name := range examples {
 			example := contentType.Examples[name]
+			if exampleRef := example.Ref; exampleRef != "" {
+				parse, err := url.Parse(exampleRef)
+				if err != nil {
+					return err
+				}
+				if refType := strings.Split(parse.Fragment[1:], "/")[1]; refType == "schemas" {
```

Comment:
I see jobs fail around here. I'm just skimming here but maybe try using the `ReadFromURIFunc`s instead of unrolling your own.

Reply:
sorry, i not found `ReadFromURIFunc` can do this, i fixed by check the ref prefix